osu.Game/Graphics/UserInterface/OsuSliderBar.cs

```diff
@@ -35,6 +35,7 @@ public class OsuSliderBar<T> : SliderBar<T>, IHasTooltip, IHasAccentColour
         private readonly Container nubContainer;
 
         public virtual string TooltipText { get; private set; }
+        public bool DisplayAsPercentage { get; set; }
```

Comment:
Should updating `DisplayAsPercentage` trigger a change of tooltip? This is quite verbose and unlikely to be used, but keeps consistency.

Reply:
I highly doubt there would be cases this property would be changed at run time and the tooltip text updating is handled here anyways https://github.com/ppy/osu/blob/a3cfeb08d41d3b4b8235b6422fc35c8d5cf16b0d/osu.Game/Graphics/UserInterface/OsuSliderBar.cs#L111
Regardless, it's for the sake of consistency

Reply:
please xmldoc